qiskit/aqua/algorithms/minimum_eigen_solvers/qaoa/qaoa.py

```diff
@@ -88,8 +89,8 @@ def __init__(self,
             p: the integer parameter p as specified in https://arxiv.org/abs/1411.4028,
                 Has a minimum valid value of 1.
             initial_state: An optional initial state to prepend the QAOA circuit with
-            mixer: the mixer Hamiltonian to evolve with. Allows support of optimizations in
-                constrained subspaces as per https://arxiv.org/abs/1709.03489
+            mixer: the mixer Hamiltonian to evolve with or a custom quantum circuit. Allows support
+                of optimizations in constrained subspaces as per https://arxiv.org/abs/1709.03489
```

Comment:
... as well as warm-starting the optimization as introduced in http://arxiv.org/abs/2009.10095

Reply:
Added.